generator-cas-app/generators/app/templates/Makefile

```
@@ -0,0 +1,16 @@
+SHELL := /usr/bin/env bash
+
+.PHONY: install_asdf_tools
+install_asdf_tools:
+	@cat .tool-versions | cut -f 1 -d ' ' | xargs -n 1 asdf plugin-add || true
+	@asdf plugin-update --all
+	@bash ~/.asdf/plugins/nodejs/bin/import-release-team-keyring
+	@#MAKELEVEL=0 is required because of https://www.postgresql.org/message-id/1118.1538056039%40sss.pgh.pa.us
+	@MAKELEVEL=0 POSTGRES_EXTRA_CONFIGURE_OPTIONS='--with-libxml' asdf install
+	@asdf reshim
+	@pip install -r requirements.txt
+	@asdf reshim
+
+.PHONY: install_dev_tools
+install_dev_tools: $(call make_help,install_dev_tools,install development tools via asdf)
```

This `make_help` function requires some functions defined in the `cas-pipeline` repo. I however found a much easier solution so make a `make help` command work:
1) add the following target at the top of the file (under the SHELL line):
```makefile
help:     ## Show this help.
	@sed -ne '/@sed/!s/## //p' $(MAKEFILE_LIST)
```

and replace the line above with this:
```suggestion
install_dev_tools: ## install development tools via asdf
```

---

That makes sense, I was wondering what was going on here, not a Makefile expert 😀